releases/index.html

```diff
@@ -106,6 +106,13 @@ <h1>Releases</h1>
                 <h2 id="repo">GenePattern Notebook Repository</h2>
                 <p>Release updates for the <a href="https://notebook.genepattern.org" target="_blank">GenePattern Notebook Repository</a>.</p>
 
+                <h3>GenePattern Notebook Repository 19.10 <em style="font-size: 0.8em; color: gray;">(October 28, 2019)</em></h3>
+                <ul>
+                    <li>Pinned public notebook tags now display an optional description.</li>
+                    <li>Updated to JupyterHub 1.0.</li>
```

Comment:
We should link out to the doc or announcement for this

Reply:
@tmtabor Sorry to be a pain, but Michael wanted us to keep linking out to things like this "JupyterHub 1.0"